src/main/scala/wordcounter/AkkaWordCounter.scala

```diff
@@ -81,17 +90,23 @@ class RoutingActor(fileRef: FileReference, listener: ActorRef) extends Actor {
   }
 }
 
+object Listener {
+  def props = Props[Listener]
```

Comment:
http://doc.akka.io/docs/akka/current/scala/actors.html#Props
`Props.apply()` is call by name. When you use it inside an Actor it's pretty easy to pass in something that will cause part of the actor's internal state to remain in scope. Wrapping the call in a call by value method gets rid of the risk. 

For the line you're commenting on, this change isn't strictly necessary.